### PR TITLE
Add Recipe: Getting Started with Jupyter Locally

### PR DESCRIPTION
**Pull request by fayvor**
Resubmitting the PR for [#87](https://github.com/ibm-granite-community/pm/issues/87)

Run recipe notebooks locally.

- Clone GC repo
- Create and activate a Virtual Environment
- Install and run Jupyter
- Open a recipe notebook in Jupyter

### CODE REVIEW

**Comment by deanwampler** [comment]
The cell that git clones the cookbook repo doesn't make sense to me, because you already have the repo cloned in order to be in the notebook (in most cases...), right?

File: recipes/Getting_Started_with_Jupyter_Locally/Getting_Started_with_Jupyter_Locally.ipynb [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "git clone git@github.com:ibm-granite-cookbooks/granite-code-cookbook.git\n",
```
Comment (deanwampler) [inline recipes/Getting_Started_with_Jupyter_Locally/Getting_Started_with_Jupyter_Locally.ipynb:30]: In addition to my other comment, all the shell commands have to start with `!`, e.g., `!git clone ...`. It appears that Jupyter allows you to do `pip install ...` without `!`, but when Ir un the other "shell cells", they fail unless I add `!`.

**Review by deanwampler — COMMENTED** [review]
Once I got to the end of this notebook, I realized that it's really more documentation than a notebook that you can run, including the initial suggestion to clone the repo.

So, I recommend just converting this to a Markdown file, which GitHub and Jupyter Labs render nicely.

File: recipes/Getting_Started_with_Jupyter_Locally/Getting_Started_with_Jupyter_Locally.ipynb [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "pip install notebook"
```
[the remark this answers is not in the document]
Comment (fayvor) [reply recipes/Getting_Started_with_Jupyter_Locally/Getting_Started_with_Jupyter_Locally.ipynb:98]: Converting to a MD file, but noted for next time.

File: recipes/Getting_Started_with_Jupyter_Locally/Getting_Started_with_Jupyter_Locally.ipynb (hunk 1 above)
Comment (fayvor) [reply recipes/Getting_Started_with_Jupyter_Locally/Getting_Started_with_Jupyter_Locally.ipynb:30]: I'll convert to a Markdown file, but noted for shell cells.

**Comment by fayvor** [comment]
> The cell that git clones the cookbook repo doesn't make sense to me, because you already have the repo cloned in order to be in the notebook (in most cases...), right?

I figured folks might just be visiting the repo online, looking to get started. This recipe is at the very beginning of the journey, so I wanted to provide sufficient context, even if they've already cloned the repo.

I think that block will make more sense in an MD file.

**Comment by fayvor** [comment]
@deanwampler this is now a markdown file. LMK how it looks to you.

**Review by deanwampler — APPROVED** [review]
Looks great! Thanks.

**Pull request merged**